pkg/handlers/authentication/okta/provider.go

```diff
@@ -116,7 +123,7 @@ func (op *Provider) AuthorizationURL(r *http.Request) (*Data, error) {
 	params := authURL.Query()
 	// Add the nonce and scope to the URL when getting ready to redirect to the login URL
 	params.Add("nonce", state)
-	params.Set("scope", "openid profile email")
+	params.Set("scope", "openid profile email okta.myAccount.profile.read")
```

Comment:
Question out of ignorance here - I know we were initially lookign throgh scopes for OIDC usage. Is this scope still necessary if using API key?

Reply:
It is not, I will correct this. Nice catch!

Reply:
Corrected and pushed